### DISCUSSION TITLE
How to detect a Vaillant/Bulex boiler in error (F28 + F54): Which fields to look for in the csv?

### DISCUSSION THREAD

**Discussion by netweaver1970** [john30/ebusd#1056] [General]
All,
my boiler has a annoying problem (probably gasvalve, since a known weakness), making in it harder in wintertime to have a reliable cold-start of the radiator heating (and even sometimes -very seldom- of the hot water heating). 
Most of the time I can continue anyhow after doing a restart, once I had to knock the valve inside to get it 'moving' again. 

I know I'll have to replace it one day, but I'm not in desperate need for it (yet). Only problem is, often I'm not home when I want to switch on the house heating.
Yesterday I have put a sonoff powr2 in series of the power so I can 'reset' the boiler remotely, using HomeAssistant. But I would like to automate it, when the boiler gives F28/F54, then reset.

Which BAI csv fields I could use to find the last errors?

Thanks.

**Comment by benoitpoirier** [comment]
Have you find a solution ?
I've currently the same problem for the winter, and before finding my Saunier Duval problem, I want to reset my boiler after a F28 error.
But when I simulate a F28 error (stop the gaz input), I've no particular field change in ebusd report.
Thanks

**Comment by benoitpoirier** [comment]
I've just found the solution.
I'm using ebusd addon for Home Assitant.
In the /etc/ebusd/ directory there is the MQTT config with mqtt-hassio.cfg.
I duplicate it in /config/ebusd/mqtt-hassio.cfg and rfernce it in the ebusd addon parameters.

After that the trick ;)
Customise your config file at the line :
filter-name = status|temp|humidity|yield|count|energy|power|runtime|hours|starts|mode|curve|^load$|^party$|sensor|timer
change it with 
filter-name = state|status|temp|humidity|yield|count|energy|power|runtime|hours|starts|mode|curve|^load$|^party$|sensor|timer

And restart !
After that, you will find the Statenumber.
If Statenumber = 98 , it's mean you have a F28 error.

To test, you can read manually Statenumber with
ebusctl read Statenumber
This command force push value in MQTT.

That's it !
